lib/logging/logging.dart

```diff
@@ -331,11 +343,24 @@ class LogDetailsUI extends CoreElement {
   LogData get data => _data;
 
   set data(LogData value) {
+    // TODO(devoncarew): Reset the vertical scroll value if any.
```

Comment:
This is probably `content.element.scrollTop = 0;` but I don't have it set up on this machine to test.

Reply:
Yup, that works!